ui-asset-management/api/svc-asset-details/extension/clients/config_client.go

```diff
@@ -41,14 +42,25 @@ func LoadConfigurationDetails() *Configuration {
 		enableExtension = true
 	}
 
+	// We only want to use assume role if config dynamodb and secrets manager is in a different account
+	assumeRoleArn := os.Getenv("ASSUME_ROLE_ARN")
+	useAssumeRole := false
+	if assumeRoleArn != "" {
+		fmt.Printf("using ASSUME_ROLE_ARN to assume role: %s\n", assumeRoleArn)
+		useAssumeRole = true
+	} else {
+		fmt.Println("ASSUME_ROLE_ARN environment variable not set, defaulting to false")
```

Comment:
minor nit: I'd change the wording slightly to end in " variable not set, defaulting to existing role"

I think there are either two or three places in this change set with the same message.